tools/packaging/hosting_debian_config.json

```diff
@@ -0,0 +1,35 @@
+{
+    "maintainer_name":"Microsoft",
+    "maintainer_email": "nugetaspnet@microsoft.com",
+
+    "package_name": "dotnet-hosting",
+    "install_root": "/usr/share/dotnet",
+
+    "short_description": "Microsoft ASP.NET Core Hosting Bundle 2.0.0",
```

Comment:
Calling it `Microsoft .NET Core 2.0.0 Linux Server Hosting` since the windows hosting bundle's installer shows 
![image](https://user-images.githubusercontent.com/2030323/28634829-54081c94-71ed-11e7-939f-3bda6af1d91a.png)